docs/config/advanced.md

```diff
@@ -82,14 +82,74 @@ Filters: there are several operators that act as filters to remove results you d
 
 - `fix` allows for an expression to be displayed to the user as the suggested autofix. Can be applied with `--autofix`
 
+## Path Filtering
```

Comment:
Just a heads up, I've also been iterating on the docs here: #476. This will probably require some merging at some point.

Reply:
This is on experimental branch and for now no plans to merge it into develop fwiw @mschwager